src/commcare_cloud/commands/terraform/constants.py

```diff
@@ -28,6 +31,7 @@
 ^/a/([\w\.:-]+)/receiver/submission/?$
 ^/a/([\w\.:-]+)/settings/users/commcare/fields/$
 ^/a/([\w\.:-]+)/settings/users/commcare/upload/$
+^/a/([\w\.:-]+)/settings/users/web/upload/$
```

Comment:
@orangejenny same question as above

Reply:
Steph's offline, but I'm reasonably confident this should be here. This view was decorated with `waf_allow` in https://github.com/dimagi/commcare-hq/pull/28832/ and it's likely just an oversight that this file wasn't also updated.